libtenzir/builtins/contexts/lookup_table.cpp

```diff
@@ -276,11 +296,25 @@ class ctx final : public virtual context {
 
   /// Inspects the context.
   auto show() const -> record override {
-    return record{{"num_entries", context_entries.size()}};
+    // TODO: there's not size() function for the patricia trie, so we need to
+    // track the number of elements externally and integrate it into this
+    // output.
```

Comment:
We could track insertions/deletions and then compute the number of elements in the subnet tree.